src/rtc/rtc_s1_single_job.py

```diff
@@ -52,6 +52,8 @@
 BROWSE_IMAGE_MIN_PERCENTILE = 3
 BROWSE_IMAGE_MAX_PERCENTILE = 97
 
+FLAG_BROWSE_DUAL_POL_REPEAT_COPOL = True
```

Comment:
If I'm not wrong, I think it actually means Pauli Decomposition? If yes, please consider renaming it.

Reply:
Thank you, @seongsujeong , but Pauli decomposition is different from what we're doing here.